sapi/litespeed/lsapilib.c

```diff
@@ -2819,7 +2819,12 @@ static void lsapi_sigchild( int signal )
         if ( WIFSIGNALED( status ))
         {
             int sig_num = WTERMSIG( status );
+
+#ifdef WCOREDUMP
             int dump = WCOREDUMP( status );
+#else
+            int dump = -1;
```

Comment:
```
#ifdef WCOREDUMP
    const char *dump = WCOREDUMP( status ) ? "1" : "0";
#else
    const char *dump = "unknown";
#endif
lsapi_log("Child process with pid: %d was killed by signal: "
                     "%d, core dump: %s\n", pid, sig_num, dump );
```

Possibly clearer?

Reply:
Yeah, that does seem better. What about yes/no instead of 0/1?

Reply:
Sounds good to me!